CMakeLists.txt

```diff
@@ -133,7 +133,7 @@ endif()
 
 include(${HPCC_SOURCE_DIR}/cmake_modules/optionDefaults.cmake)
 
-if ( REMBED )
+if ( REMBED AND (NOT INCLUDE_PLUGINS) )
   # various components that are not needed if only making the R plugin
   SET(USE_LIBARCHIVE 0)
   SET(USE_APR 0)
```

Comment:
I can see libhtpasswdSecurity.so in this build (CentOS7) and in HPCC CE 6.2.14-1 for both CentOS 6 and 7. Though I didn't install. Just verify the contents of rpm packages